base/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM golang:1.12.9-stretch AS builder
+
+MAINTAINER free5GC <support@free5gc.org>
+
+ENV GO111MODULE=off
+
+RUN apt-get update
+RUN apt-get -y install gcc cmake autoconf libtool pkg-config libmnl-dev libyaml-dev
+RUN apt-get -y install netcat tcpdump iproute2 netbase
+RUN apt-get clean
+
+# Get Free5GC
+RUN cd $GOPATH/src \
+    && git clone https://github.com/free5gc/free5gc.git \
```

Comment:
I think `-j nproc` in

```console
./install_env.sh -j `nproc`
```
and
```console
make -j `nproc`
```
do not have any effect. I assume we can safely remove them.

Reply:
```bash
./install_env.sh -j `nproc`
```
Yes, -j does not get benefit in `install_env.sh`

But in my testing, there still get 3 min faster on adding -j to `make` and `submodule` so I think we still can left it at `make` and `submodule`

Reply:
I've updated `install_env.sh` line. But I still wonder if we keep the eval of `nproc` instead of having a static number of 33. Based on this stackoverflow [answer](https://stackoverflow.com/questions/26023395/how-to-speed-up-parallelize-downloads-of-git-submodules-using-git-clone-recu/34762036#34762036) and the shared links of git parallel fetch implementation, the number of jobs corresponds to the number of threads that would be spawned by the git process. In this case wouldn't be a better option to stick to the value returned by `nproc` since it adapts automatically to the available hardware. But in the same time, 33 is ok, since if a hardware could not go beyond/or not authorized to go beyond, it simply wont.

Anyway, too much words for a small issue :smile:

Reply:
Oh, yes. maybe using `nproc` is better due to the hardware limit.

Thanks a lot!